tsl/test/t/006_job_crash_log.pl

```diff
@@ -0,0 +1,65 @@
+# This file and its contents are licensed under the Timescale License.
+# Please see the included NOTICE for copyright information and
+# LICENSE-TIMESCALE for a copy of the license.
+
+use strict;
+use warnings;
+use PostgresNode;
+use Test::More;
+use TestLib;
+
+my $node = get_new_node('primary');
+$node->init(allows_streaming => 1);
+$node->append_conf('postgresql.conf',
+	"shared_preload_libraries = 'timescaledb'");
+$node->start;
+
+# by default PostgresNode doesn't doesn't restart after a crash
+# taken from 013_crash_restart.pl
+$node->safe_psql(
+	'postgres',
+	q[ALTER SYSTEM SET restart_after_crash = 1;
+				   ALTER SYSTEM SET log_connections = 1;
+				   SELECT pg_reload_conf();]);
+
+# create extension
+$node->safe_psql('postgres', 'CREATE EXTENSION timescaledb');
+
+# create proc to run as job
+my $query =
+  q[CREATE OR REPLACE PROCEDURE custom_proc_sleep60(jobid int, config jsonb) LANGUAGE PLPGSQL AS
+$$
+BEGIN
+  RAISE NOTICE 'im about to sleep for 60 seconds, plenty of time for you to kill me';
+  perform pg_sleep(60);
+END
+$$; ];
```

Comment:
Tip: Here docs are easier to read.
 ```suggestion
my $query = <<'END_OF_SQL';
CREATE OR REPLACE PROCEDURE custom_proc_sleep60(jobid int, config jsonb) LANGUAGE PLPGSQL AS
$$
BEGIN
  RAISE NOTICE 'im about to sleep for 60 seconds, plenty of time for you to kill me';
  perform pg_sleep(60);
END
$$;
END_OF_SQL
```

Reply:
Thank you, fixed